x-pack/plugin/eql/qa/mixed-node/src/test/java/org/elasticsearch/xpack/eql/qa/mixed_node/EqlSearchIT.java

```diff
@@ -222,6 +314,23 @@ private String randomEvent() {
         return expectedResponse;
     }
 
+    private void assertMultiValueFunctionQuery(Set<String> availableFunctions, Set<String> testedFunctions, Request request,
+        RestClient client, String functionName, String query, int[] ids) throws IOException {
+        // trivial checks on the validity of functions being used
+        assertTrue("Function [" + functionName + "] not found in the list of functions " + availableFunctions,
+            availableFunctions.contains(functionName));
```

Comment:
Isn't this check redundant? I would expect the query to fail anyway if the function is not in the registry.

Reply:
Removed it. Thanks.